zoos/zoo-model.js

```diff
@@ -0,0 +1,58 @@
+const knex = require('knex');
+
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: './data/lambda.db3',
+    },
+};
+
+const db = knex(knexConfig);
+
+module.exports = {
+    find,
+    findById,
+    add,
+    update,
+    remove,
+};
+
+function find() {
+    return  db('zoos');
+}
+
+function findById(id) {
+    return db('zoos')
+    .where({ id })
+    .first()
+}
+
+function add(zoo) {
+    return  db('zoos')
+    .insert(zoo)
+    .then( (ids) => {
+        const [id] = ids;
+
+        db('zoos')
```

Comment:
this is good code but don't forget to `return` this. Whenever you are inside a then; you have to return it; thats why you're not getting anything back on post